### PR TITLE
[bug rem size] properly display sizes

### PR DESCRIPTION
**Pull request by duggiemitchell**
Fixes https://github.com/oddbird/sassdoc-theme-herman/issues/339

Trello: https://trello.com/c/NrvQuX57

## Description
- Adds `customCSS` for `sizes` and `fonts` previews.

## Reviewer tasks:

- [ ] Pulled branch and ran code locally (or viewed on preview deployment, if applicable)
- [ ] Reproduced the issue to review in the browser
- [ ] Reviewed code
- [ ] Updated Trello card (if applicable)

### CODE REVIEW

**Comment by jgerigmeyer** [comment]
[I want Herman to properly display sizes](https://trello.com/c/NrvQuX57/12-i-want-herman-to-properly-display-sizes)

**Comment by jgerigmeyer** [comment]
@mirisuzanne Okay, fonts are including `customCSS`, and both sizes and ratios are rendering in iframes (also with `customCSS`). Passing to you for review/cleanup.

File: lib/annotations/ratios.js [hunk 1]
```diff
@@ -1,11 +1,43 @@
...
 
+const ensureSassJson = require('../utils/ensureSassJson');
+const renderIframe = require('../renderIframe');
+
 /**
  * Custom `@ratios` annotation. Accepts optional
  * map-variable name (used to access data from Sass JSON).
  */
-module.exports = () => ({
+module.exports = env => ({
```
Comment (duggiemitchell) [inline lib/annotations/ratios.js:10]: 🤦🏾 `ratios` somehow slipped by me; thinking this affected just `fonts` and `sizes`.  
(It's not clear to me the difference between `sizes` and `ratios`)

Comment (jgerigmeyer) [reply lib/annotations/ratios.js:10]: Yeah, I forgot those were distinct annotations (though very similar) also.

It definitely took me a bit to get my mind back around how this all works... The annotation (e.g. `/// @ratios`) from scss is parsed and turned into a data object, then that is passed into the `resolve` fn which can add more data (e.g. rendering markup in an iframe), and then all of that is passed through to the base nunjucks templates...

File: gulpfile.js [hunk 2]
```diff
@@ -130,21 +130,21 @@ const prettierTask = function(src, shouldLog) {
...
+//   const stream = gulp
+//     .src(src)
+//     .pipe(sasslint())
+//     .pipe(sasslint.format())
+//     .pipe(sasslint.failOnError());
+//   if (!failOnError) {
+//     stream.on('error', onError);
+//   }
+//   return stream;
+// };
```
Comment (stacyk) [inline gulpfile.js:147]: I am trying to figure out why we are commenting this out?

Comment (jgerigmeyer) [reply gulpfile.js:147]: Because the linter is broken and unsupported -- we're replacing it in #348, which is a breaking change.

**Pull request merged**